java/common/common-invoke-tables/src/main/java/sleeper/invoke/tables/InvokeForTables.java

```diff
@@ -50,6 +53,7 @@ public static void sendOneMessagePerTableByName(
     }
 
     private static void sendMessageBatch(AmazonSQS sqsClient, String queueUrl, List<TableStatus> tablesBatch) {
+        LOGGER.info("Sending table batch of {} to SQS queue {}", tablesBatch, queueUrl);
```

Comment:
It might be good to explicitly say how many tables are in the batch, and put the array at the end of the line?

Reply:
Done, that looks a lot better, thanks!